flags.go

```diff
@@ -3,6 +3,11 @@ package main
 import "github.com/codegangsta/cli"
 
 var (
+	flStore = cli.StringFlag{
+		Name:  "store",
+		Value: "/var/lib/docker/swarm/store",
```

Comment:
Not sure - I agree that it's bad to `sudo` just for that.

On the other hand, it's risky to use `$HOME` since changing user would mean losing the state (or having part of the state in `vieux` and other parts in `root`).

Thoughts?